common/djangoapps/enrollment/management/tests/test_enroll_user_in_course.py

```diff
@@ -0,0 +1,66 @@
+""" Test the change_enrollment command line script."""
+
+import ddt
+import unittest
+from django.contrib.auth.models import User
+
+from django.core.cache import cache
+from django.test import TestCase
+from django.test.utils import override_settings
+from django.conf import settings
+
+from django.core.management import call_command
+
+from enrollment.tests import fake_data_api
+from enrollment.api import get_enrollment
+
+
+@ddt.ddt
+@override_settings(ENROLLMENT_DATA_API="enrollment.tests.fake_data_api")
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class EnrollManagementCommandTest(TestCase):
+    """
+    Test the enroll_user_in_course management command
+    """
+
+    USER_EMAIL = "ralph@example.com"
+    COURSE_ID = "some/great/course"
+
+    def setUp(self):
+        super(EnrollManagementCommandTest, self).setUp()
+        fake_data_api.reset()
+        User.objects.create_user('ralph_user', self.USER_EMAIL, 'fake_pass')
+        cache.clear()
+
+    def test_enroll_user(self):
```

Comment:
Please add a test for trying to enroll a user that is already enrolled in the course.